### DISCUSSION TITLE
Error while  installing the hashids using composer

### DISCUSSION THREAD

**Discussion by w3labz** [wintercms/winter#393] [Q&A] [answered]
Tries installing the hashids and following issue comes up

```
composer require hashids/hashids
Using version ^4.1 for hashids/hashids
./composer.json has been updated
Running composer update hashids/hashids
Loading composer repositories with package information
Updating dependencies
Lock file operations: 1 install, 0 updates, 0 removals
  - Locking hashids/hashids (4.1.0)
Writing lock file
Installing dependencies from lock file (including require-dev)
Nothing to install, update or remove
1 package suggestions were added by new dependencies, use `composer suggest` to see details.
Generating autoload files

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

[RuntimeException]
Could not scan for classes inside "tests/fixtures/backend/models/UserFixture.php" which does not appear to be a file nor a folder
```

Anyone faced same issue? Pls help

Also tried removing the vendor folder and tried running composer install but same issue comes up.

**Comment by bennothommo** [comment, marked as answer]
@w3labz what happens if you remove the lock file as well (`composer.lock`)?

You can also try removing the `autoload-dev` section from `composer.json` and re-run the command, as it appears - I assume - that you have installed Winter through the Installer here. The installer does not include development or testing files.

**Comment by w3labz** [comment]
@bennothommo Thanks man it did the trick. It worked like charm.
